manager/container.go

```diff
@@ -47,7 +47,7 @@ import (
 var enableLoadReader = flag.Bool("enable_load_reader", false, "Whether to enable cpu load reader")
 var HousekeepingInterval = flag.Duration("housekeeping_interval", 1*time.Second, "Interval between container housekeepings")
 
-var cgroupPathRegExp = regexp.MustCompile(`devices[^:]*:(.*?)[,;$]`)
+var cgroupPathRegExp = regexp.MustCompile(`memory[^:]*:(.*?)[,;$]`)
```

Comment:
can you add a comment explaining why we use `memory`?  I'm not actually sure why we use `devices`, but it would be nice to know why we chose that.